src/chains/tests/test_views.py

```diff
@@ -195,6 +196,7 @@ def test_json_payload_format(self) -> None:
             "recommendedMasterCopyVersion": chain.recommended_master_copy_version,
             "disabledWallets": [],
             "features": [],
+            "warning": None,
```

Comment:
We are missing a test if the warning is set.

The factory method should also be updated

Reply:
Added in [02e17b2](https://github.com/safe-global/safe-config-service/pull/688/commits/02e17b2aadef9298675f6238cfaf769509feb33c) 👍🏻